tests/microos/patterns.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Smoke pattern installation check
+# Maintainer: QA-c Team <qa-c@suse.de>
+
+use Mojo::Base qw(consoletest);
+use testapi;
+use transactional qw(trup_call process_reboot);
+use utils qw(zypper_call);
+use serial_terminal qw(select_serial_terminal);
+
+sub run {
+    shift->select_serial_terminal();
+
+    # display installed patterns
```

Comment:
maybe also displaying non installed patterns? 
I mean you could do
```
my $uninstalled_patterns = script_output "zypper -q se -t pattern -u");
record_info(uninstalled_patterns, $uninstalled_patterns);
my @available_patterns = split(/\n/, $uninstalled_patterns);
```

Or in the first record info, instead of `zypper se -t pattern -i`, we print ALL `zypper se -t pattern'` ?

Reply:
yeah, let's print all of them. Done

Reply:
ok, then you're gonna kill me for picky: `# display patterns`  :D

Reply:
ok, you were faster :)